frontend/src/lib/k8s/cluster.ts

```diff
@@ -37,6 +37,16 @@ export interface KubeMetadata {
   selfLink: string;
   labels?: StringDict;
   annotations?: StringDict;
+  ownerReferences?: KubeOwnerReference[];
+}
+
+export interface KubeOwnerReference {
```

Comment:
Documenting the interface fields would be helpful. 

```typescript
  /** */
  apiVersion: string;
```

Reply:
This is basically a copy from the [K8s reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/). I don't think there's a lot of value by adding the docs. Maybe adding a link + context in a comment at the top should be enough.